### PR TITLE
Refactoring of the language component #70

### PR DESCRIPTION
**Pull request by GrayJacketStudios**
# Pull Request

## 💬 Context
Added the switch language options in more views, so the users in android and ios can see them
#70 

## 👩🏼‍🎨 Changes

- Extract the Language Selector showed in the Login view, to its own atom component.
- Created unit tests for the Language Selector Component
- Added the component to the next views:
> - Login
> - Define a master password
> - Enter master password

## 💬 Other comments

Optionally, any other relevant comments can be included here.

## 📸 Images

imagen
imagen

### CODE REVIEW

File: src/components/atoms/LanguageSelector/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import i18n from 'i18next';
+import styled from 'styled-components';
+import useTranslationHook from '../../../hooks/useTranslation/index.js';
+import AtomButtonLink from '../ButtonLink/index.js';
+
+const LanguageSelector = styled.div`
+    margin-top: 40px;
+`;
+
+const AtomLanguageSelector = () => {
```
Comment (oegea) [inline src/components/atoms/LanguageSelector/index.js:31]: As this component depends on AtomButtonLink, it should be a molecule instead of an atom.

Comment (GrayJacketStudios) [reply src/components/atoms/LanguageSelector/index.js:31]: Noted!
Working on it

Comment (GrayJacketStudios) [reply src/components/atoms/LanguageSelector/index.js]: solved in commit 4606e38

**Review by oegea — APPROVED** [review]
Great work! 👏

**Pull request merged**